script.js

```diff
@@ -0,0 +1,197 @@
+let todoList = [];
+let doneList = [];
+
+// 입력된 값을 todo list에 추가
+const addTodoItem = () => {
+  const todoContent = document.querySelector(".todo-input").value;
+  if (todoContent) paintTodoItem(todoContent);
+};
+
+// 화면에 todo item을 추가
+const paintTodoItem = (text) => {
+  const todoId = new Date().valueOf();
+  const newTodo = document.createElement("li");
+  const newTodoText = document.createElement("span");
+  const todoDel = document.createElement("img");
+
+  // todo item 텍스트 추가
+  newTodoText.innerText = text;
+  newTodoText.style.cursor = "pointer";
+  newTodoText.style.fontSize = "15px";
+  newTodoText.style.paddingLeft = "5px";
+  newTodoText.addEventListener("dblclick", toggleTodoToDone);
+
+  // todo item 삭제 버튼 추가
+  todoDel.setAttribute("src", "img/bin.png");
+  todoDel.style.width = "14px";
+  todoDel.style.paddingLeft = "5px";
+  todoDel.style.cursor = "pointer";
+  todoDel.addEventListener("click", deleteTodoItem);
+
+  // li에 item 추가
+  newTodo.setAttribute("id", todoId);
+  newTodo.style.marginBottom = "13px";
+  newTodo.appendChild(newTodoText);
+  newTodo.appendChild(todoDel);
+
+  // todo list에 item 추가
+  const todoObj = {
+    id: todoId,
+    text,
+  };
+  todoList.push(todoObj);
```

Comment:
```suggestion
  todoList = [...todoList, todoObj]
```

객체에 변형을 가하는 것 보단 새롭게 할당하는 방식으로 객체를 변경하는 것이 좋습니다.